bip-0373.mediawiki

```diff
@@ -52,9 +52,11 @@ The new per-input types are defined as follows:
 | rowspan="2"| 0, 2
 |-
 | The MuSig2 aggregate public key (compressed) <ref>'''Why the compressed aggregate public key instead of x-only?'''
-BIP 32 requires public keys to include their evenness byte. Aggregate public keys are expected to be
-derived from, following [[bip-0328.mediawiki|BIP 328]], and therefore will
-need to include the evenness. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
+BIP 32 public keys can be derived from a BIP 327 MuSig2 aggregate public key (see: [[bip-0328.mediawiki|BIP 328]]).
+But since BIP 32 requires public keys to include their evenness byte, BIP 327 MuSig2 aggregate public keys must
+include their evenness byte as well.
+
+Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
```

Comment:
Not sure about adding the inoperative newlines within this footnote, feel free to ignore.

Reply:
Wasn't sure which was preferred: making space to preserve the unedited text (easiest diff to review) or reformatting/reflowing the entire paragraph with new line breaks (nicest final form).

Reply:
The extra blank line does not render as a paragraph break, if that was what was intended.

Reply:
Yes, probably clearer not to add them.

Reply:
* Updated to remove invisible paragraph break.
* Raw text reflowed to fit within 120 col width to be more tidy (even though has no impact on how it's actually rendered).